.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "c:\\Users\\simon\\Documents\\Mes documents\\Doctorat\\SkewLinearAlgebra.jl"
+}
```

Comment:
I don't think you intended to add this file?

Reply:
Indeed, I didn't see it. Thanks!